Document/0x05b-Basic-Security_Testing.md

```diff
@@ -119,6 +175,8 @@ $ adb push cacert.cer /sdcard/
 
 You should then be prompted to confirm installation of the certificate (you'll also be asked to set a device PIN if you haven't already).
 
+For Android 7 and above follow the same methods presented in Real Device testing section.
```

Comment:
For Android 7 and above follow the same procedure described in the "Bypassing the Network Security Configuration" section.